File: android/app/src/main/java/com/zhengqi/app/data/model/Article.kt
package com.zhengqi.app.data.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "articles")
data class Article(
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
    val title: String,
    val summary: String,
    val content: String,
    val category: String,
    val isFavorite: Boolean = false,
    val readCount: Int = 0,
    val createdAt: Long = System.currentTimeMillis()
)

File: ios/ZhengQi/Models/Article.swift
import Foundation
import SwiftData

@Model
final class Article {
    var id: UUID
    var title: String
    var summary: String
    var content: String
    var category: String
    var isFavorite: Bool
    var readCount: Int
    var createdAt: Date

    init(id: UUID = UUID(), title: String, summary: String, content: String, category: String = "修心", isFavorite: Bool = false, readCount: Int = 0, createdAt: Date = Date()) {
        self.id = id
        self.title = title
        self.summary = summary
        self.content = content
        self.category = category
        self.isFavorite = isFavorite
        self.readCount = readCount
        self.createdAt = createdAt
    }
}